src/gripper_service/src/gripper_service.cpp

```diff
@@ -0,0 +1,107 @@
+/*Copyright 2023 Brookhaven National Laboratory
+BSD 3 Clause License. See LICENSE.txt for details.*/
+#include <gripper_service/gripper_service.hpp>
+
+using namespace std::placeholders;
+
+GripperService::GripperService()
+: node_(std::make_shared<rclcpp::Node>("gripper_server_node")),
+  gripper_(kComPort, kSlaveID)
+{
+  RCLCPP_INFO(node_->get_logger(), "Activate the gripper ...");
+  // Clear the registers
+  gripper_.deactivateGripper();
+  std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
Look into if there is a better way to check this instead of forcing a sleep.

Reply:
gripper API does not provide checks to see gripper status. Force sleeps are removed for time being and pending tests on the robotic hardware.
https://github.com/maffettone/erobs/blob/9f0d358470cda04c943f454cd38ed3fd825efaa5/src/gripper_service/src/gripper_service.cpp#L13-L16